coral-base-button/src/scripts/BaseButton.js

```diff
@@ -457,6 +457,11 @@ const BaseButton = (superClass) => class extends BaseLabellable(superClass) {
     // Update autoAriaLabel as well
     iconElement.autoAriaLabel = iconAutoAriaLabelValue;
 
+    // Accessibility fix
+    // If the wrapping <button> has either a "title=" or an "icon=" attribute
+    // passes the value down to the <coral-icon> as "alt="
+    (this.title || this.icon) && (iconElement.alt = this.title || this.icon);
```

Comment:
I think this case is only valid when button itself does not have any text content in it, secondly this code can be avoided by setting iconAutoAriaLabel to Icon.autoAriaLabel.ON

Reply:
the icon attribute doesn't seem like it would ever be satisfactory alt text. Looking at the list of [icons](https://developer.adobe.com/experience-manager/reference-materials/6-5/coral-ui/coralui3/Coral.Icon.html#availableIcons) these are not suitable as alt text. Perhaps as a fallback you could space separate instead of camel case - but even then these wouldn't be localized so it would still seem like a bad idea.